rskj-core/src/main/java/co/rsk/rpc/modules/trace/TraceTransformer.java

```diff
@@ -168,7 +176,14 @@ public static TraceAction toAction(TraceType traceType, InvokeData invoke, CallT
         if (traceType == TraceType.CALL) {
             input = TypeConverter.toUnformattedJsonHex(invoke.getDataCopy(DataWord.ZERO, invoke.getDataSize()));;
             value = TypeConverter.toQuantityJsonHex(callValue.getData());
-            to = new RskAddress(invoke.getOwnerAddress().getLast20Bytes()).toJsonString();
+
+            if (callType == CallType.DELEGATECALL) {
+                to = new RskAddress(codeAddress.getLast20Bytes()).toJsonString();
```

Comment:
Here you don't check `codeAddress` for `null` when calling `codeAddress.getLast20Bytes()`. But there are some places where `null` is being passed for the `codeAddress` argument, like [here](https://github.com/rsksmart/rskj/pull/1284/files#diff-bd261f93e538742227ec20b4908a721dR391), so looks like this param is `nullable`.

I understand that it could be the case (btw.. is it ? ) that according to the logic `codeAddress` should never be `null` when `callType == CallType.DELEGATECALL`, but I guess it should be explicitly specified here in the code and properly handled if for some odd reason `codeAddress` is `null` when `callType == CallType.DELEGATECALL`

Reply:
Done